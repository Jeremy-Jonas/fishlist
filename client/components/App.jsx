import React from 'react'

import FishList from './FishList'

const App = () => <FishList />

export default App

