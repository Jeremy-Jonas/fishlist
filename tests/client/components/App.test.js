import React from 'react'
import {shallow} from 'enzyme'

import App from '../../../client/components/App'

test('<App />', () => {
  const expected = '<BrowserRouter />'
  const wrapper = shallow(<App />)
  expect(wrapper.text()).toMatch(expected)
})
