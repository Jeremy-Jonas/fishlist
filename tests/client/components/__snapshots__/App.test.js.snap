// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches the last snapshot 1`] = `
<BrowserRouter>
  <React.Fragment>
    <Route
      component={[Function]}
      path="/"
    />
    <Container
      style={
        Object {
          "marginTop": 75,
        }
      }
    >
      <Switch>
        <Route
          component={[Function]}
          exact={true}
          path="/"
        />
        <Route
          component={[Function]}
          path="/new"
        />
        <Route
          component={[Function]}
          path="/fish/:id"
        />
        <Route
          component={[Function]}
          path="/register"
        />
      </Switch>
    </Container>
  </React.Fragment>
</BrowserRouter>
`;
